components/layout_2020/style_ext.rs

```diff
@@ -365,6 +367,25 @@ impl ComputedValuesExt for ComputedValues {
         }
     }
 
+    /// Return true if this style is a normal block and establishes
+    /// a new blockformatting context.
+    fn block_establish_formatting_context(&self) -> bool {
```

Comment:
```suggestion
    /// a new block formatting context.
    fn establishes_block_formatting_context(&self) -> bool {
```

Let's keep the name consistent with the other methods here.
Nit: `blockformatting` →`block formatting`